superset/sqllab/command.py

```diff
@@ -122,25 +110,7 @@ def run(  # pylint: disable=too-many-statements,useless-suppression
                 "status": status,
                 "payload": self._execution_context_convertor.serialize_payload(),
             }
-        except SupersetErrorsException as ex:
-            if all(ex.error_type in USER_CLIENT_ERRORS for ex in ex.errors):
-                raise SupersetSyntaxErrorException(ex.errors) from ex
-            raise ex
-        except SupersetException as ex:
-            if ex.error_type in USER_CLIENT_ERRORS:
-                raise SupersetSyntaxErrorException(
-                    [
-                        SupersetError(
-                            message=ex.message,
-                            error_type=ex.error_type,
-                            level=ErrorLevel.ERROR,
-                        )
-                    ]
-                ) from ex
-            raise ex
         except Exception as ex:
-            query_id = query.id if query else None
-            logger.exception("Query %d: %s", query_id, type(ex))
```

Comment:
would it be useful to still log query id here or no?

Reply:
I don't think having the id helps us debug the issue at all tbh, with that information we can't do anything else.

the important thing is that we log the exception here (https://github.com/apache/superset/blob/9edb581f103d3e29b40d1c814cbe935a3d36f486/superset/views/base.py#L202) once it gets bubbled up

Reply:
got it!